docs/how-to-guides/develop-a-tool/use-file-path-as-tool-input.md

```diff
@@ -0,0 +1,98 @@
+# Use FilePath as tool input
+
+In this document, we will guide you through the detailed process of how to use FilePath as tool input with 'hello-world' tool example.
+
+## Prerequisites
+
+Since FilePath is still in test currently, you need to install private version SDK and Extension.
+
+1. Download private version promptflow SDK
+
+    ```bash
+    pip install "promptflow==0.0.106113823" --extra-index-url https://azuremlsdktestpypi.azureedge.net/test-promptflow/
+    ```
+
+2. Download this [VSCode Extension](https://aka.ms/pfvsctest), install from VSIX:
+
+    ![install from vsix](../../media/how-to-guides/develop-a-tool/install_from_vsix.png)
+
+## Use FilePath as tool input in 'hello-world' tool
+
+For details of how to create and use custom tool, please refer to [Custom tool package creation and usage](../how-to-create-and-use-your-own-tool-package.md).
+
+1. Create a tool project hello-world-proj with `--case file_input` to generate demo case of file_path as tool input.
+
+    ```bash
+    python <path-to-scripts>\tool\generate_tool_package_template.py --case file_path --destination hello-world-proj --package-name hello-world --tool-name hello_world_tool --function-name get_sqrt    
+    ```
+
+2. Go to hello-world-proj folder, check your tool code.
+
+   The demo use a sematic_kernel case, refer to [run native code with Semantic Kernel](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/plugins/native-functions/using-the-skfunction-decorator?tabs=python).
+
+   ![update custom tool](../../media/how-to-guides/develop-a-tool/generate_custom_tool_with_file_path.png)
+
+3. Build and install tool package in VSCode Extension, according to [Custom tool package creation and usage](../how-to-create-and-use-your-own-tool-package.md).
```

Comment:
how-to-create-and-use-your-own-tool-package.md is not exist

Reply:
Step 3 and the rest can be another section: Use tool with FilePath input in VSCode Extension